website/docs/components/form/primitives/partials/code/how-to-use.md

```diff
@@ -56,6 +58,8 @@ If the helper text needs to contain more than just text, use the block form of t
 
 !!! Warning
 
+**Accessibility consideration**
```

Comment:
[question] should this be a "Accessibility alert", since it's a "warning"? or the "warning" should be an "info"?
```suggestion
**Accessibility alert**
```